libs/components/storybook/src/lib/preview-wrapper/preview-wrapper.component.ts

```diff
@@ -24,8 +24,8 @@ import { Subscription } from 'rxjs';
 export class PreviewWrapperComponent implements OnInit, OnDestroy {
   @Input()
   public set theme(value: 'default' | 'modern-light' | 'modern-dark') {
```

Comment:
Input values can always be `undefined`.

Reply:
Sigh. I actually had fixed this one and then messed it up on a merge. Fixed.